samples/core/pom.xml

```diff
@@ -209,32 +213,39 @@
       <groupId>javax.el</groupId>
       <artifactId>javax.el-api</artifactId>
     </dependency>
-
+    
```

Comment:
careful with adding spaces where not needed because then the reviewer has to struggle thinking if it is relevant or not